docs/launch.md

```diff
@@ -85,6 +85,27 @@ html_theme_options = {
 This will create a new Deepnote project every time you click the launch button.
 ```
 
+## RetroLite
+
+To add [RetroLite](https://jupyterlite-sphinx.readthedocs.io/en/latest/retrolite.html) links to your page, add the following configuration:
+
+```python
+html_theme_options = {
+    ...
+    "launch_buttons": {
+        "retrolite_url": "./lite/retro/notebooks/"
+    },
+    ...
+}
+```
+
+```{warning}
+This requires installation of [`jupyterlite-sphinx`](https://jupyterlite-sphinx.readthedocs.io/) and the publication of a RetroLite environment as part of your published Jupyter Book environment.
+
+You __must__ also publish the Jupyter Book / JupyterLite environemt via a webserver in order to access the JupyterLite environment.
+
+*As of `jupyterlite-sphinx==0.4.7, you will have to manually ensure that notenook files are copied over to the `_build/html/lite/files` directory in order to work with them in JupyterLite.*
+```
```

Comment:
````suggestion
## JupyterLite and RetroLite

```{warning}
JupyterLite and RetroLite are experimental, and their behavior and configuration may change over time!
Please provide feedback and suggestions.
```

[JupyterLite](https://jupyterlite.readthedocs.io/) allows you to run a Jupyter environment entirely in the browser via [WebAssembly](https://webassembly.org/) and [Pyodide](https://pyodide.org/en/stable/).

To use JupyterLite in your launch buttons, you'll first need to take these steps:

1. **Install [`jupyterlite-sphinx`](https://jupyterlite-sphinx.readthedocs.io/)**. And configure it to publish a JupyterLab environment as part of your published Jupyter Book environment by [following the installation instructions](https://jupyterlite-sphinx.readthedocs.io/en/latest/installation.html).
2. **Configure JupyterLite Sphinx to use this folder**. You can configure JupyterLite Sphinx to look for notebooks in a specified directory. Put the notebooks you wish to expose in that directory, and [follow these configuration instructions](https://jupyterlite-sphinx.readthedocs.io/en/latest/configuration.html#jupyterlite-content).
```

There are two different interfaces that you can activate with JupyterLite, each is described below.

### JupyterLab via JupyterLite

To add JupyterLab via JupyterLite to your launch buttons, use the following configuration:

```python
html_theme_options = {
    ...
    "launch_buttons": {
        "jupyterlite_url": "./lite/lab/notebooks/"
    },
    ...
}
```

Where `notebooks/` is a folder with a collection of Jupyter Notebooks you'd like to serve with JupyterLite.

### RetroLab via JupyterLite

RetroLab mimics the classic Jupyter Notebook interface, but using JupyterLab components.
It will be the default Notebook interface in the near future.

To add RetroLab via JupyterLite to your launch buttons, use the following configuration:

```python
html_theme_options = {
    ...
    "launch_buttons": {
        "jupyterlite_url": "./lite/retro/notebooks/"
    },
    ...
}
```
````

Reply:
Some suggestions to flesh this out a bit more, since most people won't know how to use JupyterLite or what it is

Reply:
the URL should be absolute (`/lite/retro/notebooks/`, without the . prefix) for it to work. This was fixed in my PR